cfme/intelligence/chargeback.py

```diff
@@ -215,6 +215,20 @@ def create(self):
         view.flash.assert_success_message('Chargeback Rate "{}" was added'.format(
             self.description))
 
+    def copy(self, *args, **kwargs):
+        navigate_to(self, 'Copy')
+        new_rate = ComputeRate(*args, **kwargs)
+        add_view = self.appliance.browser.create_view(AddComputeChargebackView)
```

Comment:
**Required**
Creating a view here isn't necessary, the 'Copy' destination has `VIEW` set to `AddComputeChagebackView`, so you can set your view when you navigate.

Should you need to create a view object, it should be done with `self.create_view()` within any class that inherits Navigateable.

```
new_rate = ComputeRate()
view = navigate_to(self, 'Copy')
view.fill_with({stuff})